cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller_test.go

```diff
@@ -416,9 +416,9 @@ func createTestConfigs(specs ...testSpec) []*testConfig {
 						"template": map[string]interface{}{
 							"spec": map[string]interface{}{
 								"infrastructureRef": map[string]interface{}{
-									"apiVersion": "infrastructure.cluster.x-k8s.io/v1beta1",
-									"kind":       machineTemplateKind,
-									"name":       "TestMachineTemplate",
+									"apiGroup": "infrastructure.cluster.x-k8s.io",
```

Comment:
We probably want test coverage for both cases as autoscaler should work with v1beta1 (apiVersion) and v1beta2 (apiGroup)